src/remediations/remediations.queries.js

```diff
@@ -8,6 +8,7 @@ const db = require('../db');
 const {NULL_NAME_VALUE} = require('./models/remediation');
 const _ = require('lodash');
 const trace = require('../util/trace');
+const dispatcher_impl = require('../connectors/dispatcher/impl');
```

Comment:
Don't pin this to impl - it'll break future integration tests for this.

const dispatcher = require('../connectors/dispatcher');